### PR TITLE
Updated integration test to work for latest version with static file …

### PR DESCRIPTION
**Pull request by andykuang1**
…handling. Also renamed expected_output to expected_echo_output

### CODE REVIEW

File: integration_test.sh [hunk 1]
```diff
@@ -1,20 +1,33 @@
-#!/bin/bash
```
Comment (mckennagalvin) [inline integration_test.sh]: I think this should be left in

File: integration_test.sh [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
+diff expected_echo_output test_output_echo > diff_echo_output
+EXIT_STATUS=$?
+diff www/default.html test_output_static1 > diff_static1_output
+EXIT_STATUS=$?
+diff pics/ducks.jpg test_output_static2_jpg > diff_static2_jpg_output
+EXIT_STATUS=$?
+diff pics/giphy.gif test_output_static2_gif > diff_static2_gif_output
+EXIT_STATUS=$?
+diff pics/mario.jpeg test_output_static2_jpeg > diff_static2_jpeg_output
 EXIT_STATUS=$?
```
Comment (mckennagalvin) [inline integration_test.sh]: so I think the way this is written currently the if statement which follows will only check the success of the last command that's executed... we want to check whether any of the commands fail

File: integration_test.sh [hunk 3]
```diff
@@ -1,21 +1,35 @@
...
 ./webserver test_config &>/dev/null &
 
 # Send a request to the server using curl
 echo "Sending a curl request"
-curl -i -s localhost:8080 > test_output
+curl -i -s localhost:8080/echo > test_output_echo
+curl -s localhost:8080/static1/default.html > test_output_static1
+curl -s localhost:8080/static2/ducks.jpg > test_output_static2_jpg
+curl -s localhost:8080/static2/giphy.gif > test_output_static2_gif
+curl -s localhost:8080/static2/mario.jpeg > test_output_static2_jpeg
```
Comment (mattdrag) [inline integration_test.sh:21]: Nice, good that you tested all files. We need to make sure to update this with future files.

**Comment by mattdrag** [comment]
looks good

**Pull request merged**